fv3core/stencils/fv_dynamics.py

```diff
@@ -495,6 +505,8 @@ def fv_dynamics(
         state["atmosphere_hybrid_a_coordinate"],
         state["atmosphere_hybrid_b_coordinate"],
         state["surface_geopotential"],
+        state["specific_humidity"].data,
+        state["graupel_mixing_ratio"].data,
```

Comment:
one of these days we'll make it more seamless to always use quantity or gt4pystorage

Reply:
It seems like an implementation detail that should be removed from users view indeed.